### PR TITLE
chore: up deps

### PR DESCRIPTION
**Pull request by Szymon-dziewonski**
### Description

---

Update of dependencies 

Closing: #issue-number

### Type of change

---

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

---

_none_

### Screenshots

---

_none_

### Checklist:

---

- [ ] My code follows the style guidelines of this project
- [ ] My changes generate no new warnings
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation (remove if not needed)
- [ ] I have updated the changelog (remove if not needed)
- [ ] I have added tests (remove if not needed)

### CODE REVIEW

File: apps/server/tsconfig.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "composite": false,
     "declaration": false,
     "declarationMap": false,
+    "module": "ESNext",
```
Comment (grixu) [inline apps/server/tsconfig.json:8]: Shouldn't we use `NodeNext` on middleware?

File: apps/server/package.json [hunk 2]
```diff
@@ -5,19 +5,21 @@
...
     "dotenv": "^16.0.3",
-    "nodemon": "^2.0.19",
+    "nodemon": "^3.1.0",
     "ts-node-dev": "^2.0.0"
   },
   "devDependencies": {
-    "@types/express": "^4.17.13"
+    "@types/express": "4.17.21",
+    "ts-node": "^10.9.2",
+    "vite-node": "^2.0.2"
```
Comment (Razz21) [inline apps/server/package.json:23]: Why update to vite? Is `ts-node` used anywhere?

**Review by grixu — COMMENTED** [review]
Good work @Szymon-dziewonski 👏

File: apps/web/package.json [hunk 3]
```diff
@@ -15,47 +15,47 @@
...
     "@testing-library/dom": "^8.19.0",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
     "@types/lodash-es": "^4.17.7",
-    "@types/node": "18.15.11",
-    "@types/react": "18.0.37",
-    "@types/react-dom": "18.0.11",
-    "@types/react-transition-group": "^4.4.5",
+    "@types/node": "20.14.10",
```
Comment (Razz21) [inline apps/web/package.json:46]: All GH workflows there are configured to node 18. SFs are fixed to 18.17 or 18.19 I guess.

File: apps/web/package.json [hunk 4]
```diff
@@ -15,47 +15,47 @@
...
     "@vue-storefront/storefront-boilerplate-sdk": "~0.1.0",
-    "classnames": "^2.3.2",
+    "classnames": "^2.5.1",
     "cross-env": "^7.0.3",
     "i18next": "^22.4.15",
     "lodash-es": "^4.17.21",
-    "next": "13.3.0",
-    "next-i18next": "^13.2.2",
-    "next-pwa": "^5.6.0",
+    "next": "14.2.3",
```
Comment (Razz21) [inline apps/web/package.json:27]: no more next13 boilerplate 😄

File: apps/server/package.json (hunk 2 above)
Comment (Szymon-dziewonski) [reply apps/server/package.json:23]: becuase it doesnt work, I didnt wanted to spend much more time on it, even on your PR which you merged with failing CI wasnt working

File: apps/web/package.json (hunk 4 above)
Comment (Szymon-dziewonski) [reply apps/web/package.json:27]: yep, but if we want to have free security issues project there is no way

File: apps/web/package.json (hunk 3 above)
Comment (Szymon-dziewonski) [reply apps/web/package.json]: https://github.com/vuestorefront/storefront-next13-boilerplate/pull/90/commits/dc23839f8f9eae5febfdd86470566c3b8f834c53

File: apps/server/tsconfig.json (hunk 1 above)
Comment (Szymon-dziewonski) [reply apps/server/tsconfig.json]: good catch, I was fiddeling with tsconfig too much to make it work and i left it like that, here's change https://github.com/vuestorefront/storefront-next13-boilerplate/pull/90/commits/734486809ea82b72c316ddc2706069b6fa7831f2

**Review by grixu — APPROVED** [review]

**Review by Razz21 — APPROVED** [review]

**Pull request merged**